docs/lineage/gravitino-server-lineage.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Gravitino server Lineage support"
+slug: /lineage/gravitino-server-lineage
+keyword: Gravitino OpenLineage
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Gravitino server provides a pluginable lineage framework to receive, process, and sink OpenLineage events. By leveraging this, you could do custom process for the lineage event and sink to your dedicated systems.
+
+## Lineage Configuration
+
+| Configuration item                            | Description                                                                                                                                                                                           | Default value                                          | Required | Since Version |
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|----------|---------------|
+| `gravitino.lineage.source`                    | The name of lineage event source. The default `http` event source will .                                                                                                                              | http                                                   | No       | 0.9.0         |
```

Comment:
`http` cannot be a source, it can be a type of source.

"The default `http` event source will ." <-- incomplete sentence.

Reply:
removed